webui/react/src/components/LogViewer/LogViewer.tsx

```diff
@@ -458,22 +456,28 @@ const LogViewerCore: React.FC<Props> = ({
       bodyNoPadding
       bodyScroll
       divider
-      hideTitle={props.hideTitle}
       maxHeight
       options={logViewerOptions}
       title={props.title}>
       <div className={css.base} ref={baseRef}>
```

Comment:
There are parts in here that can probably be simplified but will take a little bit of time to fiddle to ensure nothing significant breaks. The baseRef specifically is needed however, because the scroll buttons are absolutely positioned and the parent element (baseRef) has to have `position: relative` defined to ensure the absolute positioning of the scroll buttons are contained within the parent element.